plugin.py

```diff
@@ -299,6 +302,8 @@ def refresh_layers(self):
                             required_layer.layer = layer
                             self.iface.legendInterface().setLayerVisible(
                                 layer, True)
+                            if self.crs:
+                                layer.setCrs(self.crs)
```

Comment:
> ok. But just to be sure how does the selected CRS affect the database operations

@NyakudyaA here, we will set the crs to the selected crs as soon as the layers are added by the plugin to the canvas.